Add Docs CG charter (#1)
w3c-cg/charter-proposals: docs-cg.md

# W3C Documentation CG Charter

## Mission

The mission of the W3C Documentation Community Group (W3C Docs CG) is to ensure web developers and designers worldwide have the best information available so they can build things on the web platform.

## Scope of work

> This Community will:
> 
> - Plan and write new documentation for web platform technologies.
>   - Make sure reference docs (“the book of web platform features”) are complete.
>   - Identify needs for Guides, How-Tos, and Tutorials and create them.
> - Work with specification authors to define best practices for web developer docs, code examples, and information architecture.
> - Maintain and keep documentation up-to-date using modern approaches like docs-as-code and develop tools to create feedback loops between standards and web developer documentation.
> - Collaborate with other horizontal groups such as [WebDX](https://www.w3.org/community/webdx/), [Security](https://www.w3.org/mission/security/), [Privacy](https://www.w3.org/mission/privacy/), [Accessibility](https://www.w3.org/mission/accessibility/), [Internationalization](https://www.w3.org/mission/internationalization/), [Sustainability](https://www.w3.org/community/sustyweb/) and integrate their findings and best practices into docs.
> - Create a map of documentation sources for web developers.
>   - Identify reliable sources of information.
>   - Invite them to be part of the conversations in the CG.
> - Drive adoption (or deprecation!) of web platform technologies.
>   - Identify web platform features where sparse docs prevent adoption and understanding.
>   - Interpret developer surveys (such as the [MDN short survey on web Developer’s experience with Web Security](https://github.com/web-platform-dx/developer-research/blob/main/mdn-short-surveys/2023-05-15-security-dx/interpretation.md) and understand the role of documentation in identified shortcomings.
> - Coordinate horizontal efforts to get outdated docs updated by, e.g. helping reach out to the right docs owners.
> - Act as a hub for when docs-related efforts/changes/projects appear, so people can get these amplified/coordinated with the relevant stakeholders.

Reviewer: ```suggestion
This community group will:

- Plan and write new documentation for web platform technologies.
  - Make sure reference documentation (“the book of web platform features”) is complete.
  - Identify needs for guides, how-tos, and tutorials, and create them.
- Work with specification authors to define best practices for web developer documentation, code examples, and information architecture.
- Maintain and keep documentation up-to-date using modern approaches like docs-as-code
- Develop tools to create feedback loops between standards and web developer documentation.
- Coordinate documentation efforts: Collaborate with other W3C community groups, such as [WebDX](https://www.w3.org/community/webdx/), [Security](https://www.w3.org/mission/security/), [Privacy](https://www.w3.org/mission/privacy/), [Accessibility](https://www.w3.org/mission/accessibility/), [Internationalization](https://www.w3.org/mission/internationalization/), [Sustainability](https://www.w3.org/community/sustyweb/) and integrate their findings and best practices into documentation.
- Be a resource for resources: map documentation sources for web developers.
  - Identify reliable sources of information.
  - Invite them to be part of the conversations in this community group.
- Drive adoption (or deprecation) of web platform technologies:
  - Identify web platform features where sparse docs prevent adoption and understanding.
  - Interpret developer surveys (such as the [MDN short survey on web developer’s experience with Web Security](https://github.com/web-platform-dx/developer-research/blob/main/mdn-short-surveys/2023-05-15-security-dx/interpretation.md) and understand the role of documentation in identified shortcomings.
- Ensure documentation relevance: coordinate horizontal efforts to get outdated docs updated such as by helping reach out to the right docs owners.
- Be a hub for web documentation: when docs-related efforts, changes, and/or projects appear, coordinate with the relevant stakeholders to get these amplified.
```

Reviewer: this doesn't change the meaning; just clarification, bullets, grammar, and nits.

# W3C Documentation CG Charter

## Mission

> The mission of the W3C Documentation Community Group (W3C Docs CG) is to ensure web developers and designers worldwide have the best information available so they can build things on the web platform. The Docs CG focuses on the art of technical writing and information architecture across the Web ecosystem (W3C, WHATWG, ECMA, IETF, etc.).

Reviewer 2: I think coordinating horizontal efforts is just as important for this group than focusing on the art of writing and IA. So I'd actually add this to the mission statement as well.

If I'm a web dev vaguely interested in the CG, I fear I'd be a bit intimidated by the mission and might not continue reading. I'd think: I'm not a writer, this isn't for me.

So I think this intro/mission could do a bit of a better job representing more of the scope of the work, and appealing to all of the target audience.

In fact, maybe the mission can be shorter:

```suggestion
The mission of the W3C Documentation Community Group (W3C Docs CG) is to ensure web developers and designers worldwide have the best information available so they can build things on the web platform.
```

## Scope of work

This Community will:

- Plan and write new documentation for web platform technologies.
  - Make sure reference docs (“the book of web platform features”) are complete.
  - Identify needs for Guides, How-Tos, and Tutorials and create them.
- Work with specification authors to define best practices for web developer docs, code examples, and information architecture.
- Maintain and keep documentation up-to-date using modern approaches like docs-as-code and develop tools to create feedback loops between standards and web developer documentation.

> - Collaborate with other horizontal groups such as WebDX, Security, Privacy, Accessibility, Internationalization, Sustainability and integrate their findings and best practices into docs.

Reviewer 2: Can we link to the relevant group's homepage when possible?

- Create a map of documentation sources for web developers.
  - Identify reliable sources of information.
  - Invite them to be part of the conversations in the CG.
- Drive adoption (or deprecation!) of web platform technologies.
  - Identify web platform features where sparse docs prevent adoption and understanding.
  - Interpret developer surveys (such as the [MDN short survey on web Developer’s experience with Web Security](https://github.com/web-platform-dx/developer-research/blob/main/mdn-short-surveys/2023-05-15-security-dx/interpretation.md) and understand the role of documentation in identified shortcomings.
- Coordinate horizontal efforts to get outdated docs updated by, e.g. helping reach out to the right docs owners.
- Act as a hub for when docs-related efforts/changes/projects appear, so people can get these amplified/coordinated with the relevant stakeholders.

## Key deliverables

- Write and maintain docs in [mdn/content](https://github.com/mdn/content). See the [backlog](https://openwebdocs.github.io/web-docs-backlog/) for missing reference pages.

> - Identify best practices for web developer documentation.

Reviewer 2: I love this. If the CG could deliver a set of best practices for how to document the web, in such a way that anyone can use them, that would be an giant win.

- Drive adoption and understanding of web technologies.

> - Act as a hub to coordinate documentation projects.

Reviewer 2: This last one isn't a deliverable per say. It's a thing the group does.

Author: Removed!

## Out of scope

The group may produce Community Group Reports within the scope of this charter but not Specifications.

## Dependencies or liaisons

> - Groups

Reviewer 2: Can we also link these items to their corresponding webpages here too?

  - Open Web Docs
  - Web Developer Experience (WebDX) Community Group

>   - Webref & spec tooling maintainers

Reviewer 3: The [Spec Editors CG](https://www.w3.org/community/speced-cg/) maintains some of these tools and probably deserves to appear in the list. But then other spec authoring tools come to mind, including [Wattsi](https://github.com/whatwg/wattsi), used for the HTML spec, specific ones used in TC39 and IETF, or [SpecTec](https://webassembly.org/news/2025-03-27-spectec/) used for WebAssembly.

  - SWAG Community Group
  - TAG (especially related to “explainers”)
  - OWASP docs
  - [OSTIF](https://ostif.org/)
  - Write The Docs community
  - ARIA Practices Guide Taskforce (a subgroup of ARIA WG) who owns/maintains [APG](https://www.w3.org/WAI/ARIA/apg/).
- Docs sites
  - MDN Web Docs
  - CanIUse
  - web.dev
  - learn.microsoft.com
  - devdocs.io
  - Freecodecamp
  - w3schools
  - Popular front-end sites/blogs which also document the web, for example: [css-tricks.com](https://css-tricks.com/almanac/pseudo-selectors/)
  - Translated docs, for example: [Doka: web docs for Russian speaking developers](https://doka.guide/)

## Audience

Who is this CG for?

> - Technical Writers to talk amongst themselves.

Reviewer 2: ```suggestion
- Technical writers to collaborate.
```

"talk amongst themselves" makes it sound a bit too much like a closed group who doesn't want others to join.

> - Technical Writers to talk amongst themselves.

Reviewer 4: ```suggestion
- Technical writers and other documentation practitioners to collaborate.
```

Not all technical documentation creators are writers. There are technical editors, content strategists, and more.

- Web platform docs sites to share best practices.

> - Specification authors and implementers to give and to get feedback on docs and specs.

Reviewer 3: I suppose the intent is not to invite spec authors to come to the group and ask for general dev feedback on their spec, but more to get feedback on the spec from a documentation perspective.

```suggestion
- Specification authors and implementers to give feedback on docs and get feedback from technical writers and docs sites on specs.
```

Reviewer 4: ```suggestion
- Specification authors and implementers to give and receive feedback on documentation and specs.
```

See previous comment re: role naming. Otherwise, I think @tidoust's suggestion is great.

> - Web developers to help identify needs in understanding web platform technologies.

Reviewer 4: ```suggestion
- Web developers to identify and request support to better the understanding of web platform technologies.
```
